src/main/java/com/checkout/SdkAuthorizationType.java

```diff
@@ -0,0 +1,8 @@
+package com.checkout;
+
+public enum SdkAuthorizationType {
```

Comment:
There are some operations that only supports OAuth. Marketplace for example, is there any plan to just keep a OAUTH type?

Reply:
That's a good point. And that raises another question, what happens if a client is configured with static keys, and tries to use an OAUTH only operation?

Definitely will add that option along with a custom exception that explicitly mentions such incompatibility.